plugins/database/redshift/redshift.go

```diff
@@ -478,3 +478,16 @@ $$;`)
 
 	return dbplugin.DeleteUserResponse{}, nil
 }
+
+// quoteIdentifier quotes an "identifier" (e.g. a table or a column name) to be used as part of an SQL statement.
+//
+// This is a copy of the same function as found in lib/pq (https://github.com/lib/pq/blob/v1.10.4/conn.go#L1640)
+// and was added as part of our pq -> pgx migration. The pgx package doesn't expose a similar function at this
```

Comment:
also here about licensing

Reply:
Moved both helpers to the sdk package and attached the copyright and license in [cc7abda](https://github.com/hashicorp/vault/pull/15343/commits/cc7abda403d06f29677e9ad9556c0da070048005).